packages/builder/src/components/userInterface/ComponentDropdownMenu.svelte

```diff
@@ -0,0 +1,107 @@
+<script>
+import { MoreIcon } from "components/common/Icons"
+import { store } from "builderStore"
+import { getComponentDefinition } from "builderStore/store"
+import ConfirmDialog from "components/common/ConfirmDialog.svelte"
+import { last } from "lodash/fp"
+import UIkit from "uikit"
+
+export let component
+
+let confirmDeleteDialog
+let dropdownEl
+
+$: dropdown = UIkit.dropdown(dropdownEl, { mode: "click", offset: 0, pos: "bottom-right", "delay-hide": 0, animation: false });
+$: dropdown && UIkit.util.on(dropdown, "shown", () => hidden = false)
+$: noChildrenAllowed = !component || getComponentDefinition($store, component._component).children === false
+$: noPaste = !$store.componentToPaste || $store.componentToPaste._id === component._id
+
+const lastPartOfName = c => (c ? last(c._component.split("/")) : "")
+
+const hideDropdown = () => {
+  dropdown.hide()
+}
+
+</script>
+
+<div class="root" on:click|stopPropagation={()  => {}}>
+  <button>
+    <MoreIcon />
+  </button>
+  <ul class="menu"  bind:this={dropdownEl} on:click={hideDropdown}>
+    <li on:click={() => confirmDeleteDialog.show()}>Delete</li>
```

Comment:
We make use of the `svelte-modal` package now for dialogs - not a huge deal for now but we will be wanting to get rid of anywhere the UIKit modals get used in future